src/native/managed/cdacreader/src/Legacy/SOSDacImpl.cs

```diff
@@ -844,7 +844,6 @@ int ISOSDacInterface.GetModuleData(ulong moduleAddr, DacpModuleData* data)
             data->metadataSize = readOnlyMetadata.Size;
 
             data->LoaderAllocator = contract.GetLoaderAllocator(handle);
-            data->ThunkHeap = contract.GetThunkHeap(handle);
```

Comment:
Do we need to set it to zero to avoid uninitialized return values?

(The C++ DAC implementation does `ZeroMemory(ModuleData,sizeof(DacpModuleData));` upfront to set everything to zero, but I do not see the same thing here.)